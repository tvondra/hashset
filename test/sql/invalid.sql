SELECT '{1,2s}'::int4hashset;
