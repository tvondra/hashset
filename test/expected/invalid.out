SELECT '{1,2s}'::int4hashset;
ERROR:  unexpected character "s" in hashset input
LINE 1: SELECT '{1,2s}'::int4hashset;
               ^
